config/config.go

```diff
@@ -86,6 +86,11 @@ func (c *Config) FillupWithDefaults() error {
 }
 
 func (c *Config) SetupNydusBinaryPaths() error {
+	// when DaemonMode = none, nydusd and nydus-image are not required
```

Comment:
`when using DaemonMode = none, nydusd and nydus-image binaries are not required`.

Reply:
fixed in https://github.com/containerd/nydus-snapshotter/pull/57